docs/reactions.md

```diff
@@ -364,9 +364,11 @@ Number of milliseconds that can be used to throttle the effect function. If zero
 
 Set a limited amount of time that `when` will wait for. If the deadline passes, `when` will reject / throw.
 
-### `signal` _(when)_
+### `signal`
 
-An AbortSignal object instance; allows you to abort waiting for the reaction via an AbortController. This will also cause the returned promise to reject with an error "WHEN_ABORTED". This option is ignored when using an effect function, and only applies with the promised based version.
+An AbortSignal object instance
+for `when` allows you to abort waiting for the reaction via an AbortController. This will also cause the returned promise to reject with an error "WHEN_ABORTED". This option is ignored when using an effect function, and only applies with the promised based version.
+for `autorun` and `reaction` allow you to dispose the reaction via the AbortController.
```

Comment:
Replace with
```
An AbortSignal object instance; can be used as an alternative method for disposal.<br>
`when`: This option is available only for the promise based version. When aborted, the promise rejects with the "WHEN_ABORTED" error.
```
Feel free to fix grammar or typos if necessary.

Reply:
When reading:
> `when`: This option is available only 

it seems like it's only for `when`

Reply:
You mean like the whole thing? Btw `when` delegates to `autorun`, can't we now support the non-promise based version as well?

Reply:
done, the non-promised version won't throw an error? as it won't get caught...